dev/docker-compose.yaml

```diff
@@ -141,22 +141,22 @@ services:
       - SERVER_SECRET=secret
       - MONGO_URL=${MONGO_URL}
       - 'MONGO_OPTIONS={"appName":"front","maxPoolSize":1}'
-      - ACCOUNTS_URL=http://localhost:3000
+      - ACCOUNTS_URL=http://host.docker.internal:3000
       - UPLOAD_URL=/files
       - ELASTIC_URL=http://host.docker.internal:9200
-      - GMAIL_URL=http://localhost:8088
-      - CALENDAR_URL=http://localhost:8095
-      - TELEGRAM_URL=http://localhost:8086
-      - REKONI_URL=http://localhost:4004
-      - COLLABORATOR_URL=ws://localhost:3078
+      - GMAIL_URL=http://host.docker.internal:8088
```

Comment:
May be we name it huly.localhost? If any name is possible?

Reply:
It's possible but I think it's more common to use `host.docker.internal` as it's a well-known domain resolvable within default network created in Docker w/o any additional configurations of the containers. Some docker applications even automatically append host.docker.internal 127.0.0.1 to /etc/hosts on your host machine when you install them.

Reply:
ok.